package.json

```diff
@@ -29,7 +29,7 @@
 	"license": "GPL-3.0",
 	"dependencies": {
 		"babel-polyfill": "=6.23.0",
-		"bitcore-mnemonic": "LiskHQ/bitcore-mnemonic#v1.2.5",
+		"bip39": "=2.4.0",
```

Comment:
Do we not want to fork this ourselves?

Reply:
We will probably do so in the future. Especially when we think about adding a new language to the wordlists (e.g. german). But that will probably not happen until lisk-js 1.1. Therefore for now we use the published version from bitcoinjs